.github/ISSUE_TEMPLATE.md

```diff
@@ -23,15 +23,15 @@
 ## Steps to reproduce the issue
 
 <!--
-Please try to re-create the issue using our Codesandbox template available here:
-https://codesandbox.io/s/x2mjypo6pp
+Please try to re-create the issue using our CodeSandbox template available here:
+https://codesandbox.io/s/github/IBM/carbon-components-react/tree/master/examples/helloworld
```

Comment:
Could we do this in a subsequent PR just to make sure it works before switching the issue template?

Reply:
We can create a new PR for changing the issue template, but it should be identical to https://codesandbox.io/s/github/emyarod/carbon-components-react/tree/updated-codesandbox-example/examples/playground, except the new link will point to `.../IBM/carbon-components-react/tree/master/...` instead

but I've reverted the URL change in the issue template and changed the name from `helloworld` to `playground`

Reply:
Totally agreed that it should be identical, just want to manually verify before changing the issue template just as a sanity check 👍